site/docs/webrtc/webhooks.md

```diff
@@ -0,0 +1,65 @@
+---
+id: webhooks
+title: Webhooks
+slug: /webrtc/webhooks
+description: A brief description of the Webhooks registered by WebRTC API calls.
+keywords:
+  - bandwidth
+  - webrtc
+  - video
+  - webhook
+image: ../../static/img/bandwidth-logo.png
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Subscribing to Webhooks
+
+The only API call that allows the registration of a callback / webhook is the POST to the participants resource that is used to create a new participant. The callback/webhook URL is declared in the `POST` to the `https://api.webrtc.bandwidth.com/v1/accounts/{accountId}/participants` endpoint. If the `callbackUrl` property is included in the payload, the value associated with that property is assumed to be a URL. That URL (if found to be valid) will be invoked on events that occur as part of the Participant life cycle.
+
+:::note
+Participant event webhooks are currently in Beta, and must be activated by submitting a support ticket to Bandwidth.
+:::
+
+:::note
+Participant event webhooks may change in the future. All attempts will be made to retain the syntax and semantics of the current implementation, but until Bandwidth completes the process of learning with our customers that the implementation is optimal, we cannot assure it. Bandwidth welcomes any and all comments on the implementation while in the Beta phase.
+:::
+
+## Life Cycle Events
+
+The events that will trigger invocation of the specified URL are:
+
+- 'onConnect' - establishment of the websocket connection that will allow the flow of WebRTC Media, an explicit action on the part of the websocket client.
```

Comment:
I think we should be somewhat vague in these descriptions. For example refer to the connection as a signalling connection, not a web socket connection. 

These callbacks represent individual device events (onConnect is a single device establishing an authenticated signaling connection, not the flow of media)